cassandra-analytics-core/src/main/java/org/apache/cassandra/spark/bulkwriter/TokenPartitioner.java

```diff
@@ -53,55 +54,71 @@ public class TokenPartitioner extends Partitioner {
     private final CassandraRing<RingInstance> ring;
     private final Integer numberSplits;
 
-    public TokenPartitioner(final CassandraRing<RingInstance> ring, final Integer numberSplits, final int defaultParallelism, final Integer cores) {
+    public TokenPartitioner(CassandraRing<RingInstance> ring,
+                            Integer numberSplits,
+                            int defaultParallelism,
+                            Integer cores)
+    {
         this(ring, numberSplits, defaultParallelism, cores, true);
     }
 
     @VisibleForTesting
-    public TokenPartitioner(final CassandraRing<RingInstance> ring, final Integer numberSplits, final int defaultParallelism, final Integer cores, final boolean randomize) {
+    public TokenPartitioner(CassandraRing<RingInstance> ring,
+                            Integer numberSplits,
+                            int defaultParallelism,
+                            Integer cores,
+                            boolean randomize)
+    {
         this.ring = ring;
         this.numberSplits = calculateSplits(ring, numberSplits, defaultParallelism, cores);
         setupTokenRangeMap(randomize);
-        validate();  // Intentionally not keeping this in readObject(), its enough to validate in constructor alone.
-        LOGGER.info("Partition map" + partitionMap);
+        validate();  // Intentionally not keeping this in readObject(), it's enough to validate in constructor alone
```

Comment:
good catch, although I prefer to be explicit `it is` . Feel free to ignore though
```suggestion
        validate();  // Intentionally not keeping this in readObject(), it is enough to validate in constructor alone
```